code/modules/events/stray_cargo.dm

```diff
@@ -24,7 +56,10 @@
 */
 /datum/round_event/stray_cargo/setup()
 	start_when = rand(20, 40)
-	impact_area = find_event_area()
+	if(admin_override_turf)
+		impact_area = admin_override_turf.loc
```

Comment:
```suggestion
	if(admin_override_turf)
		impact_area = get_turf(admin_override_turf)
```

Just in case we select an atom inside locker or something? Or is your reasoning that you want to hit the atom itself 100% of the time? If so, might be nicer to rename `override turf` to `override target` if you aren't explicitly handling for turfs.

Reply:
Impact area always expects an area instead of a turf, and admin_override_turf is always a turf due to an earlier get_turf proc. I've swapped it out for a get_area though which should make it more clear.